api/admin/controller/__init__.py

```diff
@@ -1709,8 +1709,10 @@ def validate_protocol(self, protocols=None):
             return UNKNOWN_PROTOCOL
 
     def _get_settings(self):
-        [protocol] = [p for p in self.protocols if p.get("name") == flask.request.form.get("protocol")]
-        return protocol.get("settings")
+        if hasattr(self, 'protocols'):
+            [protocol] = [p for p in self.protocols if p.get("name") == flask.request.form.get("protocol")]
+            return protocol.get("settings")
+        return []
```

Comment:
This last commit fixes an error when creating manual import collection, which does not have any settings. It tries to look them up using this method but self.protocols is not defined either.